installer/Migration/V5_1_0/lang-string/general.yaml

```diff
@@ -145,7 +145,7 @@ langStrings:
     unitId: preview
   -
     value: 'Accepts jpg, .png, .gif up to {fileSize}. Recommended dimensions: {width}px X {height}px'
-    unitId: accept_jpg_png_gif_upto_recomended_dimentions
+    unitId: accept_jpg_png_gif_upto_recommended_dimensions
```

Comment:
Hope that the correction of this unitId is applied in places where it was previously used as well.